package in.co.exception;

public class TestExceptionForError {
	public static void main(String[] args) {
		double div = 15/0;
		System.out.println(div);
	}

}
//this class is created only to check error